src/components/camera.js

```diff
@@ -1,6 +1,7 @@
 var registerComponent = require('../core/component').registerComponent;
 var THREE = require('../lib/three');
 var utils = require('../utils/');
+var bind = utils.function.bind;
```

Comment:
I wouldn't assume what resonates with me. It makes sense to me because the `function` utilities wrap functions, and there are various use cases of wrapping functions.